src/WebJobs.Script/Config/ScriptJobHostOptionsSetup.cs

```diff
@@ -45,12 +45,19 @@ public void Configure(ScriptJobHostOptions options)
             {
                 jobHostSection.Bind(options);
 
-                var fileLoggingMode = jobHostSection.GetSection(ConfigurationSectionNames.Logging)
-                    ?.GetValue<FileLoggingMode?>("fileLoggingMode");
+                var fileLoggingMode = jobHostSection.GetSection(ConfigurationSectionNames.Logging)?.GetValue<FileLoggingMode?>("fileLoggingMode");
                 if (fileLoggingMode != null)
                 {
                     options.FileLoggingMode = fileLoggingMode.Value;
                 }
+
+                // Default SendCanceledInvocationsToWorker to true if not provided
+                var sendCanceledInvocationsToWorker = jobHostSection.GetValue<bool?>(ConfigurationPropertyNames.SendCanceledInvocationsToWorker);
+                if (sendCanceledInvocationsToWorker is null)
```

Comment:
Do we have tests covering this setup? Doesn't seem like we're assigning the value of the resolved property if that is set.

Reply:
Added a test, code is working as expected `jobHostSection.Bind(options)` is populating the `sendCanceledInvocationsToWorker)` property and if it's still null and set the default.

Reply:
yes, bind will handle this. We should also be able to remove this code altogether if you change the default.